extensions/ql-vscode/src/databases-ui.ts

```diff
@@ -451,27 +451,46 @@ export class DatabaseUI extends DisposableObject {
   handleChooseDatabaseInternet = async (
     progress: ProgressCallback,
     token: CancellationToken
-  ): Promise<
-    DatabaseItem | undefined
-  > => {
-    return await promptImportInternetDatabase(
-      this.databaseManager,
-      this.storagePath,
-      progress,
-      token
-    );
+  ): Promise<DatabaseItem | undefined> => {
+    if (this.queryServer && await this.queryServer.cliServer.cliConstraints.supportsDatabaseUnbundle()) {
+      return await promptImportInternetDatabase(
+        this.databaseManager,
+        this.storagePath,
+        progress,
+        token,
+        this.queryServer.cliServer
+      );
+    } else {
+      return await promptImportInternetDatabase(        
+        this.databaseManager,
+        this.storagePath,
+        progress,
+        token
+      );
+    }
   };
 
   handleChooseDatabaseLgtm = async (
     progress: ProgressCallback,
     token: CancellationToken
   ): Promise<DatabaseItem | undefined> => {
-    return await promptImportLgtmDatabase(
-      this.databaseManager,
-      this.storagePath,
-      progress,
-      token
-    );
+
+    if (this.queryServer && await this.queryServer.cliServer.cliConstraints.supportsDatabaseUnbundle()) {
+      return await promptImportLgtmDatabase(
+        this.databaseManager,
+        this.storagePath,
+        progress,
+        token,
+        this.queryServer.cliServer
+      );
+    } else {
+      return await promptImportLgtmDatabase(
+        this.databaseManager,
+        this.storagePath,
+        progress,
+        token
+      );
+    }
```

Comment:
> Oops...should be the same as above. You don't need the if statement at all.
> 
> ```
>   handleChooseDatabaseLgtm = async (
>     progress: ProgressCallback,
>     token: CancellationToken
>   ): Promise<DatabaseItem | undefined> => {
>     return await promptImportLgtmDatabase(
>       this.databaseManager,
>       this.storagePath,
>       progress,
>       token,
>       this?.queryServer.cliServer
>     );
>   };
> ```
> 
> This works because you are testing for the cliServer later when you actually try to call unbundle.

the company are two they said if l contact old company l will lose all my balances is suppost they are from Europe l most want to know if the big rat win l am alone is the most biggest humillation  dont  look l never ask for money since the account start yes its ok l did a lot of mistakes drope big amount in pool and no out fuel and since my account was in investigation l was every day here or in brave and hackerone ipsos do not Answer me maybe do not submit correctly so you have the last Word l only whant to help people is that a crime or what can you tell me!!!